maxfw/utils/image_utils.py

```diff
@@ -0,0 +1,190 @@
+        self.target_mode = target_mode
+
+    def __call__(self, pic):
+        """
+        Args:
+            pic (bytestream or numpy.ndarray): Image to be converted to PIL Image.
+
+        Returns:
+            PIL Image: Image converted to PIL Image.
+
+        """
+        return F.to_pil_image(pic, self.target_mode, self.mode)
+
+
+class Normalize(object):
+    """
+    Normalize the image to a range between [0, 1].
+    """
+
+    def __call__(self, img):
+        """
+        Args:
+        img (PIL image or numpy.ndarray): Image to be normalized.
+
+        Returns:
+        numpy.ndarray: Normalized image.
+        """
+        return F.normalize(img)
+
+
+class Standardize(object):
+    """
+    Standardize the image (mean-centering and STD of 1).
+    """
+
+    def __call__(self, img):
+        """
+        Args:
+        img (PIL image or numpy.ndarray): Image to be standardized.
+
+        Returns:
+        numpy.ndarray: Standardized image.
+        """
+        return F.standardize(img)
+
+
+class Resize(object):
+    """Resize the input PIL Image to the given size.
+
+    Args:
+        size (sequence or int): Desired output size. If size is a sequence like
+            (h, w), output size will be matched to this. If size is an int,
+            smaller edge of the image will be matched to this number.
+            i.e, if height > width, then image will be rescaled to
+            (size * height / width, size)
+        interpolation (int, optional): Desired interpolation. Default is
+            ``PIL.Image.BILINEAR``
+    """
+
+    def __init__(self, size, interpolation=Image.BILINEAR):
+        assert isinstance(size, int) or (isinstance(size, Iterable) and len(size) == 2)
+        self.size = size
+        self.interpolation = interpolation
+
+    def __call__(self, img):
+        """
+        Args:
+            img (PIL Image): Image to be scaled.
+
+        Returns:
+            PIL Image: Rescaled image.
+        """
+        return F.resize(img, self.size, self.interpolation)
+
+
+class Rotate(object):
+    """
+    Rotate the input PIL Image by a given angle (counter clockwise).
+
+    Args:
+        angle (int or float): Counter clockwise angle to rotate the image by.
+    """
+
+    def __init__(self, angle):
+        self.angle = angle
+
+    def __call__(self, img):
+        """
+        Args:
+            img (PIL Image): Image to be rotated.
+
+        Returns:
+            PIL Image: Rotated image.
+        """
+        return F.rotate(img, self.angle)
+
+
+class Grayscale(object):
+    """Convert image to grayscale.
+
+    Args:
+        num_output_channels (int): (1 or 3) number of channels desired for output image
+
+    Returns:
+        PIL Image: Grayscale version of the input.
+        - If num_output_channels == 1 : returned image is single channel
+        - If num_output_channels == 3 : returned image is 3 channel with r == g == b
+
+    """
+
+    def __init__(self, num_output_channels=1):
```

Comment:
`assert num_output_channels in (1, 3)`?

Reply:
`RGBA` is 4 channels, so I'll have to include 4 as well.

Reply:
Actually, it's enforced in the underlying module. I have updated the documentation to match the 4-channel `RGBA` as well though. Thank you